From 2a3b98602a50ca253a889c94f25d56a12d774b48 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Tue, 20 Jul 2021 14:02:23 +0200
Subject: [PATCH 2/2] kmod: add mt7915e as wi-fi driver

In this commit, I add driver mt7915e for
MEDIATEK Corp. MT7915E 802.11ax [14c3:7915] [1], which we tested
and upcoming batch of Turris Omnia routers will include AX Wi-Fi card by default.
This is necessary as file kmod.list is included by base-min and we make
sure that the driver is included in medkits and in the base
installations.

[1] https://www.asiarf.com/shop/hot-sales/wifi6-4t4r-dual-bands-selectable-mpcie-card-ieee802-11ax-ac-a-b-g-n-2-4g-5ghz-aw7915-np1/
---
 kmod.list | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kmod.list b/kmod.list
index ddfee4d..81eea71 100644
--- a/kmod.list
+++ b/kmod.list
@@ -1,6 +1,7 @@
 # Wi-Fi drivers
 ath9k
 ath10k
+mt7915e
 
 # Firewall
 ipt-offload
-- 
2.30.2

